docker-services.yml

```diff
@@ -102,3 +102,8 @@ services:
       - REDIS_HOSTS=cache
     ports:
       - "8081:8081"
+  cap-client:
+    # todo: replace the url with cap/cap-client repository
+    build:
+      context: https://github.com/parths007/cap-client.git#docker
```

Comment:
Now that https://github.com/cernanalysispreservation/cap-client/pull/161 has been merge, lets update to official repo